packages/storage/src/indexed-db/index.ts

```diff
@@ -207,4 +215,23 @@ export class IndexedDb implements IndexedDbInterface {
       txs.add({ table: 'SPENDABLE_NOTES', value: n.toJson() });
     }
   }
+
+  private addNewSwaps(txs: IbdUpdates, swaps: SwapRecord[]): void {
+    for (const n of swaps) {
+      if (!n.swapCommitment?.inner) throw new Error('swapCommitment not present');
```

Comment:
Do we need this check? Looks like we can still call n.toJson() regardless

Reply:
I agree that this check is redundant, I added it only because a similar one is implemented for notes. 
Removed such checks for both swaps and notes